skills/axon/SKILL.md

```
@@ -0,0 +1,497 @@
+```
+
+Common options: `--limit`, `--domain`, `--full`, `--group`, `--collection`
+
+### Retrieve Document
+```bash
+axon retrieve <url> [options]
+```
+
+Common options: `--collection`, `-o`
+
+### Database Management
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)